src/components/CurrentTime.js

```diff
@@ -0,0 +1,18 @@
+import {Text} from 'react-native';
+import * as React from 'react';
+
+const CurrentTime = () => {
+  const currentTime = new Date();
+
+  return (
+    <Text>
+      Time:{' '}
+      {currentTime.toLocaleString('en-us', {
+        hour: 'numeric',
+        minute: '2-digit',
+      })}
+    </Text>
+  );
+};
+
+export default CurrentTime;
```

Comment:
This is my opinionated recommendation, feel free to push back:

Let's stick to named exports for components:
`export const CurrentTime = () => {...};`

One thing about default exports is that they allow for consumers to name the module as they wish when they import. This can lead to a vague understanding of what is being imported, and also of note when dealing with larger codebases, if we wanted to do a project wide refactor, change the name or something, we could possibly run into multiple instances of the module being imported with different names. TypeScript intellisense is usually better with named exports as well which is always a plus.